File: 0-9999/9000-9999/9600-9699/9655/9655.go
package main

import "fmt"

func solution(N int) string {
	turn := 1

	for {
		if N > 3 {
			N -= 3
		} else {
			N -= 1
		}

		if N == 0 {
			break
		}

		turn += 1
	}

	if turn%2 == 1 {
		return "SK"
	} else {
		return "CY"
	}
}

func main() {
	var N int
	fmt.Scanf("%d", &N)

	fmt.Println(solution(N))

}

File: 0-9999/9000-9999/9600-9699/9655/9655.py

def solution(N:int) -> int:
    
    turn = 1 
    while True:
        
        if N > 3:
            N-=3
        else:
            N-=1
        
        if N == 0:
            return 'SK' if turn%2 ==1 else 'CY'
        
        turn += 1
        
if __name__ == '__main__':
    N = int(input())
    print(solution(N=N))